.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/joyent/jsstyle.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = https://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/joyent/javascriptlint.git
```

Comment:
Any reason to not TritonDataCenter this file?  (There might be one, but I'd like to know it.)

Reply:
Oh, good catch. I had the wrong version of my script there.